tests/integration_test/services_test.py

```
@@ -33915,6 +33915,44 @@ def test_get_channel_endpoint_not_configured():
+                "page_access_token": "accessangtoken",
+                "verify_token": "tokenvld",
+                "static_comment_reply": "welcome"
+            }
+        },
+        headers={"Authorization": pytest.token_type + " " + pytest.access_token},
+    )
+    actual = response.json()
+    assert actual["success"] is True
+    assert actual["data"]
+    assert 'Channel' in actual["message"]
+    assert actual["error_code"] is 0
```

Ruff (0.12.2)

Use `==` to compare constant literals

Replace `is` with `==`

(F632)